dangerfile.ts

```diff
@@ -27,14 +27,25 @@ export default async () => {
   const hasChangedTests =
     danger.git.modified_files.filter(isTestFile).length > 0;
 
+  const isDestinationMaster = danger.github.pr.base.ref === "master";
+  const isReleaseBranch = danger.github.pr.head.ref.indexOf("release/") !== -1;
+
+  if (isDestinationMaster && !isReleaseBranch) {
+    fail(
+      "This PR has been opened against master, but the current branch is not a release one. Most likely you need to change destination branch"
+    );
+  } else if (isDestinationMaster && isReleaseBranch) {
+    warn(
+      "This PR repreents an App Store release and it should be merged only when this version has been release on the store"
```

Comment:
There's a couple of typos here:
`repreents` -> `represents`
`has been release` -> `has been released`